build/dependencies.props

```diff
@@ -0,0 +1,26 @@
+<Project>
+
+  <!-- ASP.NET Core (non-Universe builds) -->
+  <PropertyGroup>
+    <AspNetCoreFeed>https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json</AspNetCoreFeed>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExternalDependency Include="Libuv" Version="1.10.0" Source="$(AspNetCoreFeed)" />
+  </ItemGroup>
+
+  <!-- nuget.org -->
+  <PropertyGroup>
+    <DefaultNuGetFeed>https://api.nuget.org/v3/index.json</DefaultNuGetFeed>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExternalDependency Include="BenchmarkDotNet" Version="0.10.3" Source="$(DefaultNuGetFeed)"/>
```

Comment:
For now, we're only experimenting with lineups in Kestrel, which is still using 0.10.3. I'll update Kestrel to 0.10.9 as a separate step so that we unify with EF's benchmarks projects.